person_capture/gui_app.py

```
@@ -12778,6 +12783,41 @@ def _hdr_wic_intermediate_range(self) -> str:
             return "limited"
         return "full"
 
+    def _hdr_wic_prefilter_strength(self) -> float:
+        """Return the pre-WIC HDR-domain denoise strength for yuv420 WIC stills."""
+        if self._truthy_env("PC_DISABLE_HDR_WIC_PREFILTER"):
+            return 0.0
+        try:
+            default_strength = float(getattr(self.cfg, "hdr_wic_prefilter_strength", 0.0))
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)